.rubocop_fix_me.yml

```diff
@@ -201,6 +201,7 @@ Style/RedundantParentheses:
     - 'lib/scholars_archive/validators/other_affiliation_validator.rb'
```

Comment:
Could you fix these? They shouldn't be too difficult, maybe even auto-solvable (`rubocop -A`)